### PR TITLE
JWK Module + JWT Verification

### PR DESCRIPTION
**Pull request by ash-burnt**

### CODE REVIEW

**Comment by ash-burnt** [comment]
I believe this is now ready for review. We shall need to make some query modifications, similar to WebAuthN, so that the contract can call out to it

File: x/jwk/types/params.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// NewParams creates a new Params instance
+func NewParams() Params {
+	return Params{}
+}
+
+// DefaultParams returns a default set of parameters
+func DefaultParams() Params {
+	params := NewParams()
+	params.DeploymentGas = 0
+	params.TimeOffset = 5 * 1000 // default to 5 seconds, assuming 3 second block
```
Comment (ash-burnt) [inline x/jwk/types/params.go:24]: This should change to 20 seconds, from previous bad experiences

**Review by justinbarry — APPROVED** [review]
LGTM

**Review by BurntVal — APPROVED** [review]
LGTM

**Review by edjroz — APPROVED** [review]

**Pull request merged**